components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -161,4 +161,16 @@
     <string name="mozac_feature_addons_unsupported_caption_plural">%1$s add-ons</string>
     <!-- Text link to a sumo page for learning more about unsupported add-ons. -->
     <string name="mozac_feature_addons_unsupported_learn_more">Learn more</string>
+    <!-- Text that describes the last known status of the add-on updater - this status indicates when an add-on was successfully updated  -->
```

Comment:
cc: @flodolo

Reply:
These comments should be shorter, and provide the essential information. With all this copy and paste, localizers will just go through them without reading.

For example: 

> Displayed in the "Status" field for the updater when an add-on has been correctly updated.

Reply:
👍  sorry I will update it, I was trying to provide more context about where the string will be shown!